src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+
+        User exampleUser = null;
```

Comment:
The `exampleUser` variable is initialized to `null`. If the registration fails and an exception is caught, this variable will remain `null`, potentially leading to a `NullPointerException` in subsequent operations that use `exampleUser`. Consider adding a check to ensure `exampleUser` is not `null` before proceeding with operations that require a valid user.